src/main.rs

```diff
@@ -246,7 +246,7 @@ async fn runtime_upgrade_task(api: SubxtClient, tx: oneshot::Sender<Error>) {
 				log::info!(target: LOG_TARGET, "upgrade to version: {} successful", version);
 			},
 			Err(e) => {
-				log::warn!(target: LOG_TARGET, "upgrade to version: {} failed: {:?}", version, e);
+				log::debug!(target: LOG_TARGET, "upgrade to version: {} failed: {:?}", version, e);
```

Comment:
This will only fail if the applied update is the same as the current version.

Because substrate emit one at startup this will occur on every start so downgrading it debug.